data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,315 @@ mission "Coalition License Patch"
+	on accept
+		payment 200000
+
+
+
+mission "Coalition Yottrite 7"
+	name "Master Shipwright"
+	description "Bring Pichiie to <destination>, where she will join Lugglop's team in their Yottrite research."
+	passengers 1
+	landing
+	to offer
+		has "Coalition Yottrite 6: done"
+	source "Market of Gupta"
+	destination "Delve of Bloptab"
+	on offer
+		conversation
+			`You're immediately hailed by House Mallob when your ship is descending the atmosphere of <origin>. They guide you through the planet, bringing you to what looks like a massive castle-like structure, which appears to be made out of a similar material to that of Arach ships, at the base of a mountain.`
+			`	Other buildings are around as well, and you can see other ships landing close to your own. A dedicated docking area was built here, and you're almost certain the building closest to it is something of an outfitter. The atmosphere feels like that of a small village turned into an outdoor shopping mall.`
+			`	You're greeted by an Arach, who came in what you believe to be their version of a car, though you gave up on trying to guess the exact purpose of each pipe and plating, why some wheels weren't touching the ground, while pondering if you would comfortably fit inside one.`
+			`	The Arach takes a translation device out of his pocket, and formally greets you, "Honored to have you as a guest, the great House Mallob is, Captain <first> <last>."`
+			choice
+				`	"Are you the shipwright I'll be taking to Lugglop?"`
+					goto servant
+				`	"What's with this place? Why are there so many buildings all over?"`
+					goto property
+			label servant
+			`	"Mountains, no! But a servant of House Mallob, I am. Mostly as chauffeur, I work."`
+				goto mistress
+			label property
+			`	"No mere mansion, the private property of House Mallob is," he says, as he starts pointing to different areas. "A museum, there we have, detailing the history of Arachi ships. To the left, a shopping center is. To the right, House Mallob's private university is, where study the advanced methods of ship making, honor students from all over the Coalition do."`
+			label mistress
+			`	He asks that you go inside the vehicle, saying, "Assist Lugglop in his research, my mistress will. Meet her in the mansion, you must."`
+			`	The short trip is uneventful, with you managing to have a seat without much discomfort in the Arachi vessel, though you still don't know what the extra wheels are for.`
+			`	The entrance to the mansion isn't as castle-like as the walls, though it all still looks strangely pompous. Up close it almost looks like all the several parts and pods of an Arach Hulk or two were crumpled together into one single building.`
+			`	He guides you through the mansion, as you pass through several other servants, some rooms where he mentions that members of House Mallob are having meetings, and even meet a group of Saryd and Kimek students on the way, here to study with the shipwrights of House Mallob.`
```

Comment:
```suggestion
			`	He guides you through the mansion. You pass through several other servants, notice rooms where he mentions that members of House Mallob are having meetings, and even meet a group of Saryd and Kimek students here to study with the shipwrights of House Mallob.`
```
Various edits to fix multiple grammar issues. If you want me to elaborate, just let me know.

Reply:
didn't follow it exactly on that last part, any new issues from that?